django/contrib/admin/static/admin/css/base.css

```diff
@@ -376,7 +376,7 @@ table thead th.sorted .sortoptions {
 }
 
 table thead th.sorted .sortpriority {
-    font-size: .8em;
+    font-size: .8rem;
```

Comment:
With no opinion either way, I'm highlighting this line for eyeballing. Changing from `em` to `rem` here _may_ change the display to be larger/smaller than intended, based on the parent's font cascade; it's probably _fine_ and it is undeniably now more _consistent_ but it should be specifically checked and considered.

Reply:
@ should I commit this one ??

Reply:
Yes, we need this reverted back to `em`.